resources/Microsoft.Windows.Developer/Microsoft.Windows.Developer.psm1

```diff
@@ -135,6 +135,7 @@ if ([string]::IsNullOrEmpty($env:TestRegistryPath)) {
     $global:SearchRegistryPath = 'HKCU:\Software\Microsoft\Windows\CurrentVersion\Search\'
     $global:UACRegistryPath = 'HKLM:\Software\Microsoft\Windows\CurrentVersion\Policies\System\'
     $global:RemoteDesktopRegistryPath = 'HKLM:\SYSTEM\CurrentControlSet\Control\Terminal Server'
+    $global:LongPathsRegistryPath = 'HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem\'
```

Comment:
We need to add `$global:LongPathsRegistryPath` in the else statement to unblock the new unit tests 😉

Reply:
Yep; moving too fast for my own good